dbbackup/utils.py

```diff
@@ -417,3 +417,13 @@ def filename_generate(extension, database_name='', servername=None, content_type
         filename = REG_FILENAME_CLEAN.sub('-', filename)
         filename = filename[1:] if filename.startswith('-') else filename
     return filename
+
+
+def get_escaped_command_arg(arg):
+    if six.PY2:
+        import pipes
+        quote_func = pipes.quote
+    else:
+        import shlex
+        quote_func = shlex.quote
+    return quote_func(arg)
```

Comment:
Could you move the import on top of file, with something like:
```
try:
    from pipes import quote
except ImportError:
    from shlex import quote
```
(Maybe smater ;) )